parser/start_tokens.coffee

```diff
@@ -12,7 +12,7 @@ class StartTokens
   @expression = [
     'INTLIT', 'FLOATLIT', 'true', 'false', # Primitive literals
     'ID', '->', 'trait', 'class',          # Misc other tokens
-    @iterable, tokens.reservedWords,
+    @iterable, tokens.beginnings,
```

Comment:
Do you mind submitting an alternative PR to #50? I'm unclear what you want to do exactly.